doc/nrf/ug_matter_overview_data_model.rst

```diff
@@ -56,6 +57,42 @@ Events
    Events are a type of attributes that communicate device state changes.
    They can also be treated as historical data records of something that happened on the device in the past.
 
+Data Model example: Door Lock
+*****************************
+
+The following picture illustrates the Data Model structure of a common door lock device.
+
+.. figure:: images/matter_components_DM_doorlock.png
+   :alt: Data Model Door Lock example
+
+Each Matter node must ensure that its Endpoint 0 satisfies the requirements of the Root Node device type.
+This device type enforces the availability of clusters used in the process of :ref:`Matter commissioning <ug_matter_overview_commissioning>` and further administering of a Matter node.
+
+Besides the Root node endpoint, this device provides another endpoint, Endpoint 1, that implements the Door Lock Device Type specification.
```

Comment:
Some minor suggestion that imo would be more clear.
```suggestion
Besides the Root node endpoint, this device provides another endpoint, Endpoint 1, that implements the Door Lock Device Type functionality as defined in the Matter specification.
```

Reply:
Done